src/cms/templates/events/event_form.html

```diff
@@ -109,19 +109,27 @@ <h1 class="heading">
                         </div>
                     </li>
                 {% endfor %}
+                {% if event_translation_form.instance.id %}
+                    <li class="self-start ml-auto px-2">
+                        {% trans 'Version' %}:
+                        {{ event_translation_form.instance.version }}
+                    </li>
+                {% endif %}
             </ul>
             <div class="w-full mb-4 rounded border-2 border-blue-500 bg-white">
                 <div class="w-full p-4">
-                    {% if event_translation_form.instance.id %}
-                        <label class="inline-block">{% trans 'Version' %}:</label>
-                        {{ event_translation_form.instance.version }}<br>
-                        <label class="inline-block">{% trans 'Status' %}:</label>
-                        {{ event_translation_form.instance.get_status_display }}
-                    {% endif %}
-                    <label for="{{ event_translation_form.title.id_for_label }}" 
-                        data-slugify-url="{% url 'slugify_ajax' region_slug=region.slug language_slug=language.slug model_type='event' %}{% if event_form.instance.id %}?model_id={{ event_form.instance.id }}{% endif %}">
-                        {{ event_translation_form.title.label }}
-                    </label>
+                    <div class="flex justify-between">
+                        <label for="{{ event_translation_form.title.id_for_label }}" 
+                            data-slugify-url="{% url 'slugify_ajax' region_slug=region.slug language_slug=language.slug model_type='event' %}{% if event_form.instance.id %}?model_id={{ event_form.instance.id }}{% endif %}">
+                            {{ event_translation_form.title.label }}
+                        </label>
+                        {% if event_translation_form.instance.id %}
+                            <span class="mr-8">
+                                <label class="inline-block">{% trans 'Status' %}:</label>
+                                {{ event_translation_form.instance.get_status_display }}
+                            </span>
+                        {% endif %}
+                    </div>
                     {% render_field event_translation_form.title|add_error_class:"border-red-500" %}
                     <div id="link-container" class="flex items-center mt-4">
                         <label for="{{ event_translation_form.slug.id_for_label }}">
```

Comment:
```suggestion
                        <label for="{{ event_translation_form.slug.id_for_label }}" class="mr-2">
```
I think adding a small margin here would look better.